### DISCUSSION TITLE
How do we name packages for applications that don't have a publisher?

### DISCUSSION THREAD

**Discussion by latere-a-latere** [microsoft/winget-pkgs#704] [General]
For example, take the application [SuperTux](https://www.supertux.org/). It doesn't have a publisher and the developer is "The SuperTux Team". Would a suitable package name for this application be `SuperTux.SuperTux`?

I'm running against this issue on other applications as well, such as OpenRCT2, OpenTTD, Audacity, all projects without a clear publisher, usually open source projects.

**Comment by SoftCreatR** [comment]
`SuperTux.SuperTux` would be the correct package name.
